packages/components/datepicker/README.mdx

```diff
@@ -15,6 +15,18 @@ The Datepicker component combines text input, and calendar in dropdown.
 
 Use Datepicker to ask user's for date input in forms, modals, or filtering.
 
+## Install beta package
+
+```bash static=true
+npm install @contentful/f36-datepicker@beta
+```
```

Comment:
I guess that won't solve it for other beta/alpha packages. Maybe it makes sense to have this section in [PageContent](https://github.com/contentful/forma-36/tree/main/packages/website/components/PageContent) and make use of `isBeta`, `isAlpha` flags from FrontMatter?

Reply:
I don't see it as something with much use (yet). Maybe we can look into it when @massao has finished work on how we want to do pre-releases?

Reply:
If this is a quick fix, then 👍🏼